pages/pricing.vue

```diff
@@ -116,7 +116,7 @@ export default {
             '@type': 'Organization',
             name: 'Formester',
           },
-          highPrice: '49.00',
+          highPrice: '45.00',
```

Comment:
any reason for changing the price?

Reply:
Yes currently we have a price of $45.00 for the plan but it was not updated in the pricing schema. @anirudh-tantry

Reply:
Reverted it back monthly pricing.